### ISSUE TITLE
[Request] Personal Delivery Requester

### ISSUE THREAD

**Issue by spacehitchhiker42** [0ptera/Logistic-Train-Network#330]
**TL;DR**
A remote type inventory item that will send a delivery train to a selected track location.

**What ?**
using the delivery requester, you would click on a section of rail. A dialog would pop up, showing items available on the LTN Network. The player would select the quantities of each for delivery, and LTN would send those items to a temporary stop on the track section the player previously selected. The train would wait for empty cargo, and then return to its depot.

**Why ?**
It would be a much less tedious way to get construction supplies than setting up LTN stops for construction or hijacking an outpost LTN stop for supplies.
